testScripts/testOxfordDataset.sh.in

```diff
@@ -45,7 +45,7 @@ do
       echo "Directory output-$img exists. Skipping."
       continue
     fi
-    @EXECUTABLE_OUTPUT_PATH@/popsift-demo --log --gauss-mode vlfeat --desc-mode loop --popsift-mode --root-sift --downsampling -1 -i $imgfile
+    @CMAKE_BINARY_DIR@/@CMAKE_SYSTEM_NAME@-@CMAKE_SYSTEM_PROCESSOR@/popsift-demo --log --gauss-mode vlfeat --desc-mode loop --popsift-mode --root-sift --downsampling -1 -i $imgfile
```

Comment:
```suggestion
    @CMAKE_RUNTIME_OUTPUT_DIRECTORY@/popsift-demo --log --gauss-mode vlfeat --desc-mode loop --popsift-mode --root-sift --downsampling -1 -i $imgfile
```

`CMAKE_RUNTIME_OUTPUT_DIRECTORY` is declared in the main CMakeLists

Reply:
Ah, I had only searched in CMake docs ;-)